proto/buf.lock

```diff
@@ -5,11 +5,14 @@ deps:
     owner: envoyproxy
     repository: protoc-gen-validate
     commit: 6607b10f00ed4a3d98f906807131c44a
+    digest: shake256:acc7b2ededb2f88d296862943a003b157bdb68ec93ed13dcd8566b2d06e47993ea6daf12013b9655658aaf6bbdb141cf65bfe400ce2870f4654b0a5b45e57c09
```

Comment:
Newer versions of the buf cli will add a digest entry into `buf.lock`. This is part of an upcoming feature (sorry I don't have docs to point at). I can back these changes out, but I suggest keeping them.